src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -116,9 +116,10 @@ public function upload(): Response
         return $this->request($this->request);
     }
 
-    public function buildCreateRequest(): void
+    public function buildCreateRequest(?string $status = null): void
```

Comment:
Why this parameter is not a boolean? And the name of this parameter doesn't say much for what it is.